idaplugin/rematch/actions/match.py

```diff
@@ -34,9 +34,17 @@ def perform(self):
       i, offset = self.function_gen.next()
 
       func = instances.FunctionInstance(self.file_id, offset)
-      network.query("POST", "collab/instances/", params=func.serialize(),
-                    json=True)
-
+      try:
+        network.query("POST", "collab/instances/", params=func.serialize(),
+                      json=True)
+      except exceptions.QueryException as e:
+        data = e.response()
+        if "Invalid pk" in data['file'][0]:
+          logger('MatchAllAction').error("Something wrong happened, we can't"
+                                         " find the right database to fetch"
+                                         "infomation from.\nAre you sure you'"
+                                         "re usig the right server ?\nDid you"
+                                         "create a new database ?")
```

Comment:
"MatchAllAction" - should be module name, not function name.

"usig" -> using
can you remove spaces before the question marks? "server ?", "database ?"